src/main/java/org/fcrepo/camel/processor/ProcessorUtils.java

```diff
@@ -40,55 +42,24 @@ private ProcessorUtils() {
     }
 
     private static String trimTrailingSlash(final String path) {
-        String trimmed = path;
-        while (trimmed.endsWith("/")) {
-            trimmed = trimmed.substring(0, trimmed.length() - 1);
-        }
-        return trimmed;
-    }
-
-    /**
-     * Extract a language string suitable for Jena from a MimeType value
-     * @param contentType a MIMEType string
-     * @return a Jena-compatible language string
-     */
-    public static String langFromMimeType(final String contentType) {
-        if (contentType.equals("application/n-triples")) {
-            return "N-TRIPLE";
-        } else if (contentType.equals("text/turtle")) {
-            return "TURTLE";
-        } else if (contentType.equals("application/rdf+xml")) {
-            return "RDF/XML";
-        } else if (contentType.equals("text/rdf+n3") || contentType.equals("text/n3")) {
-            return "N3";
-        } else {
-            return null;
+        if (path.endsWith("/")) {
+            return trimTrailingSlash(path.substring(0, path.length() - 1));
```

Comment:
I was just trying to control for the case where someone had multiple slashes at the end: `http://localhost:8080/fcrepo/rest////`, but that's probably pretty unlikely -- or it should just be considered a user/configuration error that shouldn't be so gracefully handled.